Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:stretch
+MAINTAINER James McFeeters "jmcfeeters@usgs.gov"
+RUN apt-get update && apt-get install -y \
+    python3-pip \
+    nginx-light \
+    curl \
+    apt-transport-https \
+    git
+
+COPY . /app
+WORKDIR /app
+
+RUN cp DOIRootCA2.cer /etc/ssl/certs/DOIRootCA2.pem \
```

Comment:
I copy the certificate into the container with `COPY . /app`. I was under the impression that I needed to store the certificate with the system certificates, but everything seems to work fine with these lines commented out.